pkgs/by-name/uh/uhd/package.nix

```diff
@@ -32,13 +32,21 @@
   enableN320 ? true,
   enableE300 ? true,
   enableE320 ? true,
+  llvmPackages_20,
```

Comment:
Could you please move this argument upwards near `stdenv`? There is a sort of category division of the various arguments and this breaks this division.